infrastructure/airy.conf.tpl

```diff
@@ -1,6 +1,8 @@
 FB_APP_ID=changeme
 FB_APP_SECRET=changeme
 FB_WEBHOOK_SECRET=changeme
+GOOGLE_PARTNER_KEY=changeme
+GOOGLE_SA_FILE={"type":"service_account","project_id":"airy","private_key_id":"no","private_key":"nokey","client_email":"no","client_id":"no","auth_uri":"no","token_uri":"no","no":"no","client_x509_cert_url":"no"}
```

Comment:
The app validates the variable structure on startup, which is why we have to put this there until we do dynamic startups :)